System.Compiler/Reader.cs

```diff
@@ -1924,6 +1924,10 @@ private AssemblyNode.PostAssemblyLoadProcessor ReferringAssemblyPostLoad
     private TypeNodeList GetTypeParametersFor(int parentIndex, Member parent){
       GenericParamRow[] genericParameters = this.tables.GenericParamTable;
       TypeNodeList types = new TypeNodeList();
+      TypeNodeList savedTypes = this.currentTypeParameters;
+      this.currentTypeParameters = types;
+      try
+      {
       int i = 0, n = genericParameters.Length, j = n-1;

```

Comment:
:bulb: Can you correct *just* the indentation for the contents of the block? In other words, add two spaces at the beginning of each line but don't change any other formatting associated with the block.